File: elias/Problema0391/problema0391.py
matriz = []
for i in range(5):
    linhaMatriz = []
    for j in range(5):
        x = int(input(f"Digite um número para a linha {i} e coluna {j} da matriz:")) 
        linhaMatriz.append(x)
    matriz.append(linhaMatriz)
coluna = int(input("Digite o número correspondente a coluna solicitada a ser mostrada da matriz: (1 ao 5):"))
while(coluna != -1):
    print(" ")
    print("Coluna selecionada: ")
    if(coluna == 1):
        for i in range(5):
            print(matriz[i][0])
    elif(coluna == 2):
        for i in range(5):
            print(matriz[i][1])
    elif(coluna == 3):
        for i in range(5):
            print(matriz[i][2])
    elif(coluna == 4):
        for i in range(5):
            print(matriz[i][3])
    elif(coluna == 5):
        for i in range(5):
            print(matriz[i][4])
    else:
        print("Número inválido, tente novamente !")
    print("Digite o número printcorrespondente a coluna solicitada a ser mostrada da matriz: (1 ao 5):")
    coluna = int(input("Digite -1 caso queira sair. "))

File: elias/Problema0391/problema0391.java
import java.util.Scanner;

public class problema0391 {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int[][] matriz = new int[5][5];
        int coluna;
        for(int i=0;i<5;i++){
            for(int j=0;j<5;j++){
                System.out.println("Digite um número para a linha "+i+" e coluna "+j);
                matriz[i][j] = Integer.valueOf(input.nextLine()); 
            }
        }
        System.out.println("Digite o número correspondente a coluna solicitada a ser mostrada da matriz: (1 ao 5).");
        coluna = Integer.valueOf(input.nextLine());
        while(coluna != -1){
            System.out.println(" ");
            System.out.println("Coluna selecionada: ");
            if(coluna == 1){
                for (int i = 0; i < 5; i++) {
                System.out.println(matriz[i][0]);
                }
            }
            else if(coluna == 2){
                for (int i = 0; i < 5; i++) {
                System.out.println(matriz[i][1]);
                }
            }
            else if(coluna == 3){
                for (int i = 0; i < 5; i++) {
                System.out.println(matriz[i][2]);
                }
            }
            else if(coluna == 4){
                for (int i = 0; i < 5; i++) {
                System.out.println(matriz[i][3]);
                }        
            }
            else if(coluna == 5){
                for (int i = 0; i < 5; i++) {
                System.out.println(matriz[i][4]);
                }
            }
            else
                System.out.println("Número inválido, tente novamente !");
            System.out.println("Digite o número correspondente a coluna solicitada a ser mostrada da matriz: (1 ao 5). ");
            System.out.println("Digite -1 caso queira sair. ");
            coluna = Integer.valueOf(input.nextLine());
        }
    }
}